### ISSUE TITLE
Mobile User usability issues reported

### ISSUE THREAD

**Issue by kf6gpe** [flutter/website#2966]
See [this report](https://search.google.com/search-console/mobile-usability?resource_id=sc-domain:flutter.io&utm_source=wnc_10030322&utm_medium=gamma&utm_campaign=wnc_10030322&utm_content=msg_100058679&hl=en) (this link, alas, may not be visible to non-Googlers. and for that I apologize). Four issues are reported:
  * Text too small to read
  * Clickable items too close together
  * Content wider than screen
  * Viewport not set

**Comment by sfshaza2** [comment]
I am unable to see that link. @kf6gpe, can you summarize the problems?

**Comment by kf6gpe** [comment]
Drilling down, these may be non issues. It's reports of four things:

* Text too small to read
* Clickable elements too close together
* Content wider than screen
* Viewport not set.

Diving into each issue, each report appears once, but with no URLs for any of the pages offered for each report. So I'm not sure how much to heed it, actually.

**Comment by sfshaza2** [comment]
OK, please file a bug if these issues are better described in future. Closing.
